docs/background/kubernetes/gardener/hibernation.md

```diff
@@ -0,0 +1,61 @@
+---
+description: How hibernation of Gardener-based Kubernetes clusters works
+---
+
+# Putting Gardener clusters in hibernation
+
+In {{brand}}, you have the option of putting a Gardener-based Kubernetes
+[cluster in
+hibernation](../../../howto/kubernetes/gardener/hibernate-shoot-cluster.md).
+This is something you might want to do whenever you know you won't be
+needing the cluster for some time, so putting it in hibernation makes
+sense because from that time on and before you wake it up, you pay
+*less* for hosting.
+
+You may wonder why you still pay *something* after putting a cluster in
+hibernation, instead of paying nothing at all. To answer this question,
+we have to explain how hibernation actually works.
```

Comment:
Actually, I tend to think of "actually" as a fluff word that can actually be removed from most sentences, which tends to actually make the sentence more condensed and thus, actually, improves it:

```suggestion
we have to explain how hibernation works.
```

Reply:
Thanks, actually.